recipes/spyre/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.2.1" %}
+{% set commit = "ba37d1b6327afbc2c7cf8e7b8ef1d9a6fbd17d76" %}
+
+package:
+    name: spyre
+    version: {{ version }}.dev0
```

Comment:
That is ugly, but this spyre is not the same as the PyPI package and it seems like a hobby-software. The developer comes and goes. Since there has been some activity lately I will ask for a release.